apps/redi-talent-pool/src/pages/app/me/MeCompany.tsx

```diff
@@ -192,7 +183,6 @@ function isProfileComplete(
 }
 
 function SendProfileForReviewButton() {
-  const isBusy = useIsBusy()
```

Comment:
Why removing this?

Reply:
It wasn't being used. But it's better to use it and disable button when busy. Pushing that implementation.